### PR TITLE
Add several Katacoda enhancements, and update some text

### PR DESCRIPTION
**Pull request by andreaskth**

### CODE REVIEW

File: elk-analytics/index.json [hunk 1]
```diff
@@ -43,7 +43,9 @@
     "hideHiddenFiles": true,
     "showdashboard": true,
     "dashboards": [
-      {"name": "Base application", "port": 8080}
+      {"name": "Base application", "port": 8080},
+      {"name": "Elastic search", "port": 9200},
```
Comment (andreaskth) [inline elk-analytics/index.json]: 'Elasticsearch' med ett ord tror jag

File: elk-analytics/step1.md [hunk 2]
```diff
@@ -46,8 +46,8 @@ As you might have noticed, a Java error-message has appeared in the terminal. Th
 These are some endpoints available in our base application:
 * `/generate-errors` - generate 10 log messages with a log-level of `ERROR` (you can change the count by appending `/generate-errors?count=100` as a query-string)
 * `/greeting` - display a 'Hello World!'-greeting page (greet someone else by appending `/greeting?name=Katacoda` as a query-string)
-* `/calculator` - TODO Henrik
+* `/calculator` - a simple calculator that let's the user enter numbers and basic operators through a button interface. If the equals button is pressed it does some error checking on the input expression and attempts to calculate the result.
```
Comment (andreaskth) [inline elk-analytics/step1.md]: let's --> lets

**Review by andreaskth — COMMENTED** [review]
Bra ändringar! :+1:

File: elk-analytics/index.json (hunk 1 above)
Comment (hengque) [inline elk-analytics/index.json]:
```suggestion
      {"name": "Elasticsearch", "port": 9200},
```

File: elk-analytics/step1.md (hunk 2 above)
Comment (hengque) [inline elk-analytics/step1.md]:
```suggestion
* `/calculator` - a simple calculator that lets the user enter numbers and basic operators through a button interface. If the equals button is pressed it does some error checking on the input expression and attempts to calculate the result. 
```

**Review by hengque — CHANGES_REQUESTED** [review]

**Review by hengque — APPROVED** [review]

**Pull request merged**